luigi/parameter.py

```diff
@@ -959,3 +959,28 @@ def serialize(self, x):
         :param x: the value to serialize.
         """
         return json.dumps(x)
+
+class ChoiceParameter(Parameter):
+    """Parameter that restricts options to the specified set
+    All choices must be of the same type that so that parse can
+    correctly infer type
+    Variable type is specified with var_type=...
```

Comment:
Please format all docs and make sure it renders nice. See CONTRIBUTING.md and other examples in this file.